python/gps/algorithm/traj_opt/traj_opt_utils.py

```diff
@@ -85,7 +85,7 @@ def traj_distr_kl(new_mu, new_sigma, new_traj_distr, prev_traj_distr):
         )
 
     # Add up divergences across time to get total divergence.
-    return np.sum(kl_div)
+    return kl_div
```

Comment:
No, this was a mistake I made when setting up the pull request, along with several of the other comments. I had some code supporting weighted KL constraints but took it out since it wasn't helping and cluttered things. I'm making a fix for that now.